release/ray_release/cluster_manager/cluster_manager.py

```diff
@@ -108,8 +108,8 @@ def _annotate_cluster_compute(
             return cluster_compute
 
         cluster_compute = cluster_compute.copy()
-        aws = cluster_compute.get("aws", {})
-        cluster_compute["aws"] = add_tags_to_aws_config(
+        aws = cluster_compute.get("advanced_configurations_json", {})
```

Comment:
can we raise an exception here if `"aws"` is in `cluster_compute` ?

Reply:
> can we raise an exception here if `"aws"` is in `cluster_compute` ?

or we would want to handle both cases.. if we are not confident that we have changed everything in the repo.